src/test/java/VCRTest.java

```diff
@@ -87,6 +87,7 @@ public void testErase() throws Exception {
     }
 
     @Test
+    @SuppressWarnings("unused")
```

Comment:
Why would this be throwing an unused warning anyway? It's a test that's definitely being run/used.

Reply:
There is an unused variable, I am going to remove it instead of adding `SuppressWarnings("unused")`